### DISCUSSION TITLE
View Any Website as Preview [EXPLOIT]

### DISCUSSION THREAD

**Discussion by beans69** [3kh0/ext-remover#1163] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
With this exploit, you're capable of previewing any link; as long as you can right click on it, and it's HTTPS.
Steps to Activate:
1. Go to `about:flags` to go to chrome flags."
2. Enable `link-preview` - The flag should be `about:flags/#link-preview`
3. Enable flag & restart system.
4. Right click on any URL and click `Preview Link [NEW]` (If that action is not in the context - please double check the steps and try again.) and the website should pop up in the bottom right corner of the screen. You can navigate through this page fully unblocked, and to close the preview - simply close the tab that you opened the preview on.

Reddit is blocked for me, so what I'll do is find a link to Reddit, and click on it. It can be done [here](https://reddit.com/).

**Comment by S-PScripts** [comment]
chrome://flags#link-preview works but not the one you said lol

**Comment by S-PScripts** [comment]
Can't preview non-HTTPS URLPreviewing non-HTTPS URL is not allowed.
hmm

**Reply by S-PScripts** [reply]
https://roblox.com

**Reply by S-PScripts** [reply]
doesnt work, my filter is too good

**Reply by S-PScripts** [reply]
https://youtube.com

**Comment by S-PScripts** [comment]
ok so although this doesn't really work for me it actually doesn't show in history (i previewed youtube.com, went to youtube.com. not in history!)

**Comment by lukieshowie21** [comment]
bro it worked thx

**Comment by schoolexploitkid** [comment]
chrome-untrusted://crosh

**Reply by schoolexploitkid** [reply]
ugh not a link :vv

**Reply by S-PScripts** [reply]
i doubt chromeurls work with previews

**Reply by lelaw2w** [reply]
mines not blocked could you do something with this?
